components/List/index.js

```diff
@@ -0,0 +1,41 @@
+import { StyledCard } from "../Card/Card.styled";
+import Image from "next/image";
+
+export default function List({ rides }) {
+  return (
+    <ul>
+      {rides.map((ride) => (
+        <StyledCard key={ride.id}>
+          <p>
+            {ride.driver}
+            <br />
```

Comment:
I think you should not use linebreaks for styling. This should be only used within text, as far as I know. It would be better to wrap the elements in this paragraph in <span> and give them margin or padding.

Reply:
Instead of <p> and <br/> you could use <span>
`<StyledCard key={ride.id}> <span>{ride.driver}</span><span>{nextProp}</span></span><span>{nextProp}</span></span><span>{nextProp}</span>`
then yo can have a const StyledSpan = styled.span` margin-left: 2rem`;
If you want the properties to be underneath eachother, you can give the StyledCard a display: flex and flex-direction: column.
We can chat if you need a better explanation and this is still confusing.

Reply:
I think I found a solution with `<StyledSpan>`, thank you! :)